templates/admin_tools/speed_statistics_banner.html

```diff
@@ -3,9 +3,9 @@
 <body>
 <div class="alert alert-info" role="alert" id="SpeedStatisticsPopup"
      style="display: none; justify-content: space-between; align-items: center; position: relative; width: 100%; text-align: center; padding: 15px;">
-    <span class="close" style="position: absolute;
-                               top: 2px;
-                               right: 2px;">&times;
+    <span class="close" id="speed-statistics-close" style="position: absolute;
```

Comment:
Hi @eaw-pid for your next pull request, could you please update this id to pascalCase?